count-stats/src/test/java/uk/ac/ebi/eva/countstats/CountStatsApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package uk.ac.ebi.eva.countstats;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class CountStatsApplicationTests {
```

Comment:
Are tests part of another ticket?

Reply:
Added Test. 
Have added test containers for integration test. Again, this is something we can discuss in our dev meeting.